### PR TITLE
PL 80711 release notes

### PR DESCRIPTION
**Pull request by deepakbolangady**
For reviewers, previews are available.

[Platform release notes (NextGen)](https://6509dbdb5bf89928abce33e9--harness-developer.netlify.app/release-notes/platform)
[Harness SaaS Release Notes (FirstGen)](https://6509dbdb5bf89928abce33e9--harness-developer.netlify.app/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes)
[What's new](https://6509dbdb5bf89928abce33e9--harness-developer.netlify.app/release-notes/whats-new)

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — DISMISSED** [review]
Approved based on the comment to move the ticket from early access feature to fixed issue

File: release-notes/platform.md [hunk 1]
```diff
@@ -30,11 +30,66 @@ The following deprecated API endpoints will no longer be supported:
...
 
+-  The delegate Helm chart is now included in the delegate proxy configuration. You can pull the Helm chart from `https://<YOUR_LOADBALANCERURL>/storage/harness-download/delegate-helm-chart/`. (PL-39190)
+
+- The heartbeat interval that perpetual tasks use to test connectors has been increased from 10 minutes to 30 minutes. This change aims to reduce the number of errors logged due to failed heartbeats. The new heartbeat interval is used with any connectors that you create after this deployment. Tasks associated with existing connectors require migration to the new interval. Harness will migrate such perpetual tasks in all accounts in a phased manner. This activity does not require any action from you or other users of the platform. (PL-39399)
+
+- To enhance security, Harness has added settings to enable and disable Slack, Microsoft Teams, webhook, and PagerDuty notification channels at the account scope. (PL-39921)
+
+- When Harness is configured to use the AppRole ID to fetch an authentication token from HashiCorp Vault, Harness generates a large number of requests for those tokens. The volume of requests causes performance issues. (PL-40754)
```
Comment (aimurphy) [inline release-notes/platform.md:43]: under "new features" but says "this issue has been fixed"

File: release-notes/whats-new.md [hunk 2]
```diff
@@ -21,6 +21,18 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
 
 ## Latest: September 19, 2023
 
+### Harness Platform, version 80711
+
+-  The delegate Helm chart is now included in the delegate proxy configuration. You can pull the Helm chart from `https://<YOUR_LOADBALANCERURL>/storage/harness-download/delegate-helm-chart/`. (PL-39190)
```
Comment (aimurphy) [inline release-notes/whats-new.md:26]:
```suggestion
-  The delegate Helm chart is now included in the delegate proxy configuration. You can pull the Helm chart from `https://<YOUR_LOAD_BALANCER_URL>/storage/harness-download/delegate-helm-chart/`. (PL-39190)
```

**Review by aimurphy — APPROVED** [review]
approved w/ a few suggestions

File: release-notes/platform.md (hunk 1 above)
Comment (brian-f-harness) [reply release-notes/platform.md:43]: Thanks, @aimurphy. I'll move it.

File: release-notes/whats-new.md (hunk 2 above)
Comment (brian-f-harness) [reply release-notes/whats-new.md:26]: Will include this change in my commit to move the item from new features to fixed issues.

**Pull request merged**